utils/exporters/blender/addons/io_three/exporter/api/material.py

```diff
@@ -128,7 +128,7 @@ def normal_map(material):
 @_material
 def opacity(material):
     logger.debug('material.opacity(%s)', material)
-    return round(material.alpha - 1.0, 2);
+    return round( material.alpha, 2 );
```

Comment:
I think his fix is correct. I believe I made a mistake here and @WestLangley had it correct https://github.com/mrdoob/three.js/issues/5923#issuecomment-70424221